pkg/handlers/generic/lifecycle/utils/scs.go

```diff
@@ -50,3 +57,38 @@ func CreateStorageClass(
 	}
 	return &sc
 }
+
+func CreateStorageClassOnRemote(
+	ctx context.Context,
+	cl ctrlclient.Client,
+	configs []v1alpha1.StorageClassConfig,
+	cluster *clusterv1.Cluster,
+	defaultStorageConfig *v1alpha1.DefaultStorage,
+	csiProvider string,
+	provisioner v1alpha1.StorageProvisioner,
+	storageClassParameters map[string]string,
```

Comment:
nit: matches what's in `CreateStorageClass`.
```suggestion
	defaultParameters map[string]string,
```

Reply:
done